### DISCUSSION TITLE
Is pdfarranger not on pypi (anymore)?

### DISCUSSION THREAD

**Discussion by Cimbali** [pdfarranger/pdfarranger#964] [Q&A] [answered]
I’ve tried to get 1.10.0 before it’s released on my OS but pip can’t find it:
```
~> pip install --user --upgrade --force-reinstall 'pdfarranger>=1.10'
ERROR: Could not find a version that satisfies the requirement pdfarranger>=1.10 (from versions: none)
```

It also doesn’t show up on [the pypi.org search for pdfarranger](https://pypi.org/search/?q=pdfarranger).

I’ve tried looking through issues and discussions here about pypi but did not find anything relevant.

**Comment by jeromerobert** [comment, marked as answer]
It has never been on PyPI because it relies on GTK which is not (and will never be) on PyPI. And for the future there is also #860 which you already found ;-).

**Comment by jeromerobert** [comment]
@dreua What's your feeling about https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_pypi_parity ?

**Reply by Cimbali** [reply]
I guess it’s easy enough to ask PyPI to block the `pdfarranger` name and see what they reply? For pympress, I’ve made it available on pypi by checking gtk dependencies only at runtime, but not sure that’s the best solution if you already have many distribution packages covered.

**Reply by dreua** [reply]
It should already be blocked on Pypi according to https://github.com/pypi/support/issues/355

I have no strong feelings regarding this guideline, it says "should" and I assume it is more relevant for libraries. Given that the request came up only once and very late compared to Flatpak and others I'd say there is no strong demand for it.
